.github/workflows/build_and_push_production.yml

```
@@ -41,6 +42,13 @@ jobs:
         run: |
           docker push $IMAGE_NAME
 
+      - name: Deploy new image
+        run: |
+          aws lambda update-function-code \
+            --function-name $FUNCTION_NAME \
+            --image-uri $IMAGE_NAME
```

Could you suppress output here like so:
```suggestion
            --image-uri $IMAGE_NAME > /dev/null 2>&1
```
This will prevent any secret leaking since `aws lambda` commands dump a function's environment variables when they run.  Not that I think we'll ever have secrets in this function, but this has messed with me on other projects.